### PR TITLE
Save searches

### PR DESCRIPTION
**Pull request by gonzalovelasco**
Screenshot 2023-08-15 at 16 02 29
Screenshot 2023-08-16 at 15 55 16

### CODE REVIEW

**Comment by matthewwong525** [comment]
make the UI look like Google Keep, add divider and a title saying saved saerches + a button to "Create/Edit" ?

File: lib/screens/main/components/create_search_dialog.dart [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+                            stfSetState(() {});
+                          },
+                        ),
+                      ],
+                    ),
+                  );
+                },
+              ),
+            ),
+            Padding(
```
Comment (matthewwong525) [inline lib/screens/main/components/create_search_dialog.dart:133]: remove the done button

File: lib/screens/main/components/create_search_dialog.dart [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+              padding: const EdgeInsets.symmetric(horizontal: 16.0),
+              child: Row(
+                children: [
+                  Expanded(
+                    child: TextField(
+                      controller: _searchController,
+                      decoration: const InputDecoration(hintText: 'New search'),
```
Comment (matthewwong525) [inline lib/screens/main/components/create_search_dialog.dart:60]:
```suggestion
                      decoration: const InputDecoration(hintText: 'Create new search'),
```

File: lib/screens/main/components/create_search_dialog.dart [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+          mainAxisSize: MainAxisSize.min,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Padding(
+              padding: const EdgeInsets.all(16.0),
+              child: Row(
+                children: [
+                  Expanded(
+                    child: Text(
+                      'Create/Edit Searches',
```
Comment (matthewwong525) [inline lib/screens/main/components/create_search_dialog.dart:40]:
```suggestion
                      'Edit searches',
```

File: lib/screens/main/components/create_search_dialog.dart [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+              ),
+            ),
+            const SizedBox(height: 16),
+            Expanded(
+              child: ListView.builder(
+                shrinkWrap: true,
+                itemCount: widget.searches.length,
+                itemBuilder: (context, index) {
+                  String editedText = widget.searches[index];
+                  return ListTile(
```
Comment (matthewwong525) [inline lib/screens/main/components/create_search_dialog.dart:84]: Add leading search icon like in Google Keep

IMG_4FCF7D1E336E-1

**Review by matthewwong525 — CHANGES_REQUESTED** [review]
Great work! just a few small changes from me

File: lib/screens/main/components/create_search_dialog.dart (hunk 1 above)
Comment (gonzalovelasco) [reply lib/screens/main/components/create_search_dialog.dart]: Fixed 49a0cb7beb988f0e6a0251f475efd8a2b4291948

File: lib/screens/main/components/create_search_dialog.dart (hunk 4 above)
Comment (gonzalovelasco) [reply lib/screens/main/components/create_search_dialog.dart]: 183ef22ce96cf4181009785c03ae5b0ec8e98b77

**Review by matthewwong525 — APPROVED** [review]
LGTM, great work gonzalo! Merge if you're happy with it!

**Pull request merged**